composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "illuminate/support": "5.5.* || 5.6.* || 5.7.* || 5.8.*"
+        "illuminate/support": ">5.7"
     },
     "require-dev": {
         "orchestra/testbench": "~3.0",
```

Comment:
I guess Testbench needs to be updated to support Laravel 6.0 as well (Testbench `~3.0 || ^4.0`)?

However, version 4 isn't released yet.

Reply:
Good note! Updated!